app/models/screening_list/mappable.rb

```diff
@@ -33,16 +34,84 @@ def self.included(klass)
             store:   true,
           },
           properties: {
+            # name variants
             name:                    { type:     'string',
                                        analyzer: 'standard_asciifolding_nostop',
                                        fields:   {
                                          keyword: { type: 'string', analyzer: 'keyword_lowercase' } } },
 
+            name_idx:                { type:     'string',
+                                       analyzer: 'standard_asciifolding_nostop',
+                                       fields:   {
+                                         keyword: { type: 'string', analyzer: 'keyword_lowercase' } } },
+
+            name_no_common:          { type:     'string',
+                                       analyzer: 'standard_asciifolding_nostop',
+                                       fields:   {
+                                         keyword: { type: 'string', analyzer: 'keyword_lowercase' } } },
```

Comment:
What comes to mind when I see this is the [multi field type](https://www.elastic.co/guide/en/elasticsearch/reference/0.90/mapping-multi-field-type.html). I imagined the mapping would look something like this:
```
             name:                    { type:     'string',
                                        analyzer: 'standard_asciifolding_nostop',
                                        fields:   {
                                          keyword: { type: 'string', analyzer: 'keyword_lowercase' },
                                          idx: { type: 'string', analyzer: 'CUSTOM IDX ANALYZER' },
                                          no_common: { type: 'string', analyzer: 'CUSTOM NO_COMMON ANALYZER' },
                                           ... etc. ...
                                        },
             },
```
Maybe it's not possible to fashion analyzers to these specific analyzes..?